packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -600,13 +600,23 @@ class Uploader {
     const filename = this.uploadFileName
     const { client, options } = this.options.s3
 
+    function getPartSize (chunkSize) {
+      // backwards compatibility https://github.com/transloadit/uppy/pull/3511#issuecomment-1050797935
+      // requires min 5MB and max 5GB partSize
+      // todo remove this logic in the next major semver
+      if (chunkSize == null || chunkSize >= 5368709120 || chunkSize <= 5242880) return undefined
```

Comment:
```suggestion
      const fiveGiB = 5_368_709_120
      const fiveMiB = 5_242_880
      if (chunkSize == null || chunkSize >= fiveGiB || chunkSize <= fiveMiB) return undefined
```

Reply:
I think this is not supported on node10 :/ but I see node10 is no longer lts, so we should drop support for it. however that's a major version change 🤔

Reply:
Oh right 😞 yeah it's not worth it given that we want to get rid of this check in the next major anyway.